### PR TITLE
WIP: Mario/3432 Finish tutorials

### LINKED ISSUE #3432: Finish tutorials
**Is your feature request related to a problem? Please describe.**

We need to:

* Reduce dimensions of the component to half of their size (follow Zeplin measures)
* Add content
* Discuss how to trigger tutorials (i.e if balance in mainnet = 0 then show "how to get mainnet tokens?" tutorial with instructions to buy tokens from a exchange, if we are on testnet and balance = 0 then show "how to get tokens?" tutorial with instructions to get tokens from a faucet.
* Do we need to store read/closed tutorials in a cookie? That way we can avoid anoying our users.
* Do we need to emit Google Analytics events for tutorials (started, finished, closed)?

### PR DESCRIPTION
**Pull request by mariopino**
Closes #3432 

**Description:**

Finish tutorials, working in progress:

- [x] Reduce dimensions
- [x] Add contents
- [x] Add backgrounds
- [x] Include buttons to open tutorials
- [x] Testing
- [x] Import Zeplin mockup colors as CSS vars
- [x] Pending fullguide links

Maybe other PR

- [ ] Add Google Analytics events

image

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [x] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
Remember to create small PR. So merge as soon as possible and create another PR later.

File: src/components/common/InsecureModeWarning.vue [hunk 1]
```diff
@@ -1,17 +1,106 @@
 <template>
   <div>
     <p>
-      Did you know that storing your key in the browser is considered unsafe? We
-      offer a more secure way to storing your keys via our
+      Did you know that storing your key in the browser is considered unsafe?
```
Comment (jbibla) [inline src/components/common/InsecureModeWarning.vue]: i think we can delete this sentence now.

File: src/components/common/InsecureModeWarning.vue [hunk 2]
```diff
@@ -1,17 +1,106 @@
...
+    </p>
+    <p>
+      Check out our quick guide
```
Comment (jbibla) [inline src/components/common/InsecureModeWarning.vue]:
```suggestion
      Check out our guide for
```

File: src/components/common/InsecureModeWarning.vue [hunk 3]
```diff
@@ -1,17 +1,106 @@
...
+      <a href="#" @click="openTutorial()">How to Manage Your Keys</a>
```
Comment (jbibla) [inline src/components/common/InsecureModeWarning.vue]:
```suggestion
      <a href="#" @click="openTutorial()">how to safely manage your keys</a>
```

**Comment by Bitcoinera** [comment]
OK, what is going here? How can I help to speed this up @mariopino ? What is left to do?

**Comment by jbibla** [comment]
mario is waiting on me to provide him the links to the "full guides"

**Comment by jbibla** [comment]
noticed some small opportunities for improvement:
- clicking next or back resets the scroll position on the page
- last button should use the upward diagonal arrow to imply you are leaving the app
- mobile button placement is a bit funny and should be fixed

---
future:
- how do we scale these to every network from a technical perspective
- include links to exchanges, ledger store, iOS, Android, extension (i think we use v-html if we're supplying the content?)

File: src/components/common/ModalTutorial.vue [hunk 4]
```diff
@@ -51,7 +51,7 @@
                   ? `Read the full guide`
                   : `Next step`
               }}
-              <i class="material-icons arrow_forward"></i>
+              <i class="material-icons arrow_forward" :class="currentStep === steps.length ? `final-step` : ``"></i>
```
Comment (jbibla) [inline src/components/common/ModalTutorial.vue]: i would make `currentStep === steps.length` a computed property so this will not be duplicated

Comment (mariopino) [reply src/components/common/ModalTutorial.vue]: Good point!

Comment (mariopino) [reply src/components/common/ModalTutorial.vue]: This is done @jbibla ;-)

Comment (mariopino) [reply src/components/common/ModalTutorial.vue]: > noticed some small opportunities for improvement:
> 
> * [x]  clicking next or back resets the scroll position on the page
> * [ ]  last button should use the upward diagonal arrow to imply you are leaving the app
> * [ ]  mobile button placement is a bit funny and should be fixed
> 
> future:
> 
> * how do we scale these to every network from a technical perspective
> * include links to exchanges, ledger store, iOS, Android, extension

Let's merge it so we can refactor in another PR to allow including links in the content using dynamic named slots.

To scale tutorials for other networks we can:

* Refactor ModalTutorial component to use slots (so we have links!)
* Create individual tutorial components that import ModalTutorial component and have it  content inside. Easy to maintain and allow to open the same tutorial from a different component without duplicating code.
* Cause of the differences between networks, my proposal is to create a directory hierarchy for tutorials like:

```
src/components/tutorials/cosmos/governance.vue
src/components/tutorials/cosmos/staking.vue
src/components/tutorials/cosmos/keys.vue
src/components/tutorials/cosmos/tokens.vue
...
src/components/tutorials/polkadot/governance.vue
src/components/tutorials/polkadot/staking.vue
src/components/tutorials/polkadot/keys.vue
src/components/tutorials/polkadot/tokens.vue
...
```

**Comment by faboweb** [comment]
the guides from the links are not yet implemented

**Comment by jbibla** [comment]
guides are up! i'm going to edit / review again tomorrow. charlie is working on the assets.

https://lunie.io/guides/how-to-get-tokens
https://lunie.io/guides/how-to-manage-your-keys
https://lunie.io/guides/how-cosmos-governance-works
https://lunie.io/guides/how-cosmos-staking-works

**Pull request merged**